package.json

```diff
@@ -23,17 +23,17 @@
   },
   "homepage": "https://github.com/digitalbazaar/bedrock-kms",
   "dependencies": {
-    "did-io": "^0.8.2",
-    "did-method-key": "^0.7.0",
-    "did-veres-one": "^12.1.1",
-    "jsonld-signatures": "^7.0.0",
+    "@digitalbazaar/ed25519-signature-2020": "^2.0.0",
+    "bedrock-did-io": "digitalbazaar/bedrock-did-io#initial",
+    "@digitalbazaar/did-method-key": "digitalbazaar/did-method-key-js#v1.x",
+    "did-veres-one": "^13.0.0",
     "lru-cache": "^6.0.0",
-    "webkms-switch": "^3.0.0"
+    "webkms-switch": "digitalbazaar/webkms-switch#support-multiple-context-type"
   },
   "peerDependencies": {
```

Comment:
need a peerDep for `bedrock-security-context`

Reply:
addressed here c230da4